v2/boost-sdk/boost-core/claim-incentive.mdx

```diff
@@ -6,3 +6,81 @@ import SDKDisclaimer from '/snippets/sdk-disclaimer.mdx';
 import ReadParams from '/snippets/read-params.mdx';
 
 <SDKDisclaimer />
+
+## API
+
+### `claimIncentive`
+
+Claims one incentive from a given `Boost` by `boostId` and `incentiveId`
+
+```ts
+// get the boost you want to claim from
+const boost = await core.getBoost(1n)
+
+// get the incentive you want to claim off the boost object
+const incentive = boost.incentives[0]
+
+// prepare the incentive payload
+let incentivePayload
+
+if (incentive instanceof ERC20Incentive) {
+  const reward = await incentive.reward()
+  const asset = await incentive.asset()
+  const limit = await incentive.limit()
+  const strategy = await incentive.strategy()
+
+  incentivePayload = prepareERC20IncentivePayload({
+    asset,
+    reward,
+    limit,
+    strategy,
+  })
+}
+
+// encode claimData payload
+const claimDataPayload = await boost.validator.encodeClaimData({
+  signer: signer.account,
+  incentiveData: incentivePayload,
```

Comment:
this is a good example, but i think specifically for erc20incentive, this could even just be `zeroHash`

Reply:
and that part isn't clear at all because we need to expose methods on each incentive to encode data specific to that incentive